core/src/main/resources/schema/app/20191114152446_CB-1358_Rename_every_Ambari_related_DB_column_to_generic_one.sql

```diff
@@ -0,0 +1,138 @@
+-- // CB-1358 Rename every Ambari related DB column to generic one
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE cluster
+ADD COLUMN IF NOT EXISTS cloudbreakclustermanageruser varchar(255);
+UPDATE cluster
```

Comment:
it is not useful in this deployment step.
please remove all updates from all tables.